tgui/packages/tgui/interfaces/MedicalRecords.js

```diff
@@ -314,59 +374,134 @@ const MedicalRecordsViewMedical = (_properties, context) => {
+              );
+            })
+          )
+          .sort((a, b) => {
+            const i = sortOrder ? 1 : -1;
+            return a[sortId].localeCompare(b[sortId]) * i;
+          })
+          .map((vir) => (
+            <Table.Row
+              key={vir.id}
+              className={
+                'MedicalRecords__listVirus--' + vir.severity
+              }
+              onClick={() => act('vir', { vir: vir.D })}
+            >
+              <Table.Cell>
+                <Icon name="virus" /> {vir.name}
+              </Table.Cell>
+              <Table.Cell>{vir.max_stages}</Table.Cell>
+              <Table.Cell
+                color = {severities[vir.severity]}
+              >
+                {vir.severity}
+              </Table.Cell>
+            </Table.Row>
+          ))}
+      </Table>
+    </Section>
+  </Flex>
+)
+  // virus.sort((a, b) => (a.name > b.name ? 1 : -1));
+  // // here
+  // return virus.map((vir, i) => (
+  //   <Fragment key={i}>
+  //     <Button
+  //       icon="flask"
+  //       content={vir.name}
+  //       mb="0.5rem"
+  //       onClick={() => act('vir', { vir: vir.D })}
+  //     />
+  //     <br />
+  //   </Fragment>
+  // ));
 };
 
 const MedicalRecordsMedbots = (_properties, context) => {
-  const { data } = useBackend(context);
+  const { act, data } = useBackend(context);
   const { medbots } = data;
   if (medbots.length === 0) {
     return <Box color="label">There are no Medbots.</Box>;
   }
-  return medbots.map((medbot, i) => (
-    <Collapsible key={i} open title={medbot.name}>
-      <Box px="0.5rem">
-        <LabeledList>
-          <LabeledList.Item label="Location">
-            {medbot.area || 'Unknown'} ({medbot.x}, {medbot.y})
-          </LabeledList.Item>
-          <LabeledList.Item label="Status">
-            {medbot.on ? (
-              <Fragment>
-                <Box color="good">Online</Box>
-                <Box mt="0.5rem">
+  return(
+    <Flex direction="column" height="100%">
+      <Section flexGrow="1" mt="0.5rem">
+        <Table className="MedicalRecords__list">
+          <Table.Row bold>
+            <Table.Cell>Name</Table.Cell>
+            <Table.Cell>Area</Table.Cell>
+            <Table.Cell>Status</Table.Cell>
+            <Table.Cell>Chemicals</Table.Cell>
+          </Table.Row>
+          {medbots
+            .map((medbot) => (
+              <Table.Row
+                key={medbot.id}
+                className={
+                  'MedicalRecords__listMedbot--' + medbot.on
+                }
+              >
+                <Table.Cell>
+                  <Icon name="medical" /> {medbot.name}
+                </Table.Cell>
+                <Table.Cell>{medbot.area || 'Unknown'} ({medbot.x}, {medbot.y})</Table.Cell>
+                <Table.Cell>{medbot.on ? (
+                    <Box color="good">Online</Box>
+                    ) : (
+                    <Box color="average">Offline</Box>
+                  )}
```

Comment:
```suggestion
                <Table.Cell>{medbot.on 
                    ? (<Box color="good">Online</Box>) 
                    : (<Box color="average">Offline</Box>)}
```
cleaner implementation